packages/core/src/generatePolicy.js

```diff
@@ -414,7 +433,7 @@ function createModuleInspector(opts) {
       // get native modules
       native = packageToNativeModules.has(packageName)
       // skip package policy if there are no settings needed
-      if (!packages && !globals && !builtin) {
+      if (!packages && !globals && !builtin && !native) {
```

Comment:
This should have caused trouble, but I guess we were lucky that native packages always have globals listed.